### ISSUE TITLE
temp

### ISSUE THREAD

**Issue by joeflack4** [cancerDHC/ccdh-terminology-service#82]

**Comment by joeflack4** [comment]
## Expected behavior
Go to https://terminology.ccdh.io/docs#/ConceptReferences/get_concept_references or https://terminology.ccdh.io/conceptreferences, and get a response back without needing to provide any param(s).

## Actual behavior
3 params are needed.
<img width="603" alt="Screen Shot 2021-09-22 at 10 25 36 AM" src="https://user-images.githubusercontent.com/13045020/134362350-5e254efe-9785-4d50-8c50-4cd2bf23bd26.png">

**Comment by jiaola** [comment]
I agree that it might break the REST paradigm but at the same time, it's not practical to have an endpoint with no params in this case. It would list all the concept references (with pagination of course) since there could be millions of them. But it wouldn't be something a terminology service would want to provide, IMO.

**Comment by joeflack4** [comment]
@jiaola Thanks, that's the explanation I was looking for. It looks like I should go with (B) then.

@cmungall: This was something you brought up on Slack. Just pinging you here to let you know the intent of this endpoint, as Dazhi and I have discussed. I just need to update the Swagger interface to properly reflect that this endpoint requires params.
